src/main/java/uk/gov/companieshouse/company/links/consumer/ChargesStreamConsumer.java

```diff
@@ -22,7 +22,7 @@
 
 
 @Component
-public class ChargesStreamConsumer {
+public class  ChargesStreamConsumer {
```

Comment:
Is this accidental whitespace?

Reply:
It will be yes, not sure what's happened there. Will fix